packages/core/src/components/Button/button.example.html

```diff
@@ -68,19 +68,19 @@ <h6 class="preview__label">Success button</h6>
 
 <h6 class="preview__label">Transparent button</h6>
 <button type="button" class="ds-c-button ds-c-button--transparent">
-  Default
+  <span class="ds-c-button--transparent-text">Default</span>
 </button>
 <button type="button" class="ds-c-button ds-c-button--transparent ds-c-button--hover">
-  Hover
+  <span class="ds-c-button--transparent-text">Hover</span>
 </button>
 <button type="button" class="ds-c-button ds-c-button--transparent ds-c-button--active">
-  Active
+  <span class="ds-c-button--transparent-text">Active</span>
 </button>
 <button type="button" class="ds-c-button ds-c-button--transparent ds-c-button--focus">
-  Focus
+  <span class="ds-c-button--transparent-text">Focus</span>
 </button>
```

Comment:
@bernardwang Seems to be causing some display differences in both Firefox and Safari (MacOS). Haven't tested on Windows.

![Yellow background and black text on button with the yellow background not extending the full height and width.](https://user-images.githubusercontent.com/1473618/72569346-40acde00-3888-11ea-8770-0c4b3cdca1e9.png)

Reply:
Which differences are you seeing? I took a look at FF/safari/IE11 and they look similar to the screenshot you have above

Reply:
@bernardwang Sorry if I wasn't clear. See how in my screenshot above, the background doesn't any padding on the right or left. But if other browsers, it does.

Is it displaying different for you?

Reply:
I'm seeing the padding just fine in Chrome, Safari, Firefox and IE11

Reply:
@line47 @davidakennedy To clarify, the padding in the screenshot above is the same for every button. The only difference with the transparent button is that there is an additional span that is styled to have a highlighted background. https://gyazo.com/0e7d6bf55d32b93d39bc2ae02585dd1b

Reply:
I originally changed the transparent button focus because the transparent button is styled like a link, I thought it made sense for the focus styles to also be styled like a link. This brings up the point of why we provide a button that looks like a link in the first place, but my rationale is that if we do provide it, it should be consistent.

I'm willing to go back to the original styling if this doesnt make sense however. Would it make sense to get design input on this?

Before: 
<img width="674" alt="ff63e1cb0766e100892176c6510852e6" src="https://user-images.githubusercontent.com/5424713/72937471-ca4c2800-3d2e-11ea-8aa7-97922447c442.png">
After: 
<img width="590" alt="01cff3c090a8a24698ffc579b97d59ca" src="https://user-images.githubusercontent.com/5424713/72937481-d0da9f80-3d2e-11ea-8226-0fad221d6961.png">